docs/data-types/mysql.md

```diff
@@ -1,91 +1,31 @@
-
-For example, run
-
-```sh
-spanner-migration-tool data -session=mydb.session.json -source=mysql -target-profile="instance=my-spanner-instance,dbName=my-spanner-database-name" < my_mysqldump_file
-```
-
-You can also run Spanner migration tool in a schema-and-data mode, where it will perform both
-schema and data migration. This is useful for quick evaluation when source
-database size is small.
-
-```sh
-spanner-migration-tool schema-and-data -source=mysql -target-profile="instance=my-spanner-instance" < my_mysqldump_file
-```
-
-Spanner migration tool generates a report file, a schema file, and a bad-data file (if
-there are bad-data rows). You can control where these files are written by
-specifying a file prefix. For example,
-
-```sh
-spanner-migration-tool schema -prefix=mydb. -source=mysql < my_mysqldump_file
-```
-
-will write files `mydb.report.txt`, `mydb.schema.txt`, and
-`mydb.dropped.txt`. The prefix can also be a directory. For example,
-
-```sh
-spanner-migration-tool schema -prefix=~/spanner-eval-mydb/ -source=mysql < my_mysqldump_file
-```
-
-would write the files into the directory `~/spanner-eval-mydb/`. Note
-that Spanner migration tool will not create directories as it writes these files.
-
-### Directly connecting to a MySQL database
-
-In this case, Spanner migration tool connects directly to the MySQL database to retrieve
-table schema and data. Set the `-source=mysql` and corresponding source profile
-connection parameters `host`, `port`, `user`, `dbName` and `password`.
-
-For example, to perform schema conversion, run
-
-```sh
-spanner-migration-tool schema -source=mysql -source-profile="host=<>,port=<>,user=<>,dbName=<>"
-```
-
-Parameters `port` and `password` are optional. Port (`port`) defaults to `3306`
-for MySQL source. Password can be provided at the password prompt.
-
-(⚠ Deprecated ⚠) Set environment variables `MYSQLHOST`, `MYSQLPORT`,
-`MYSQLUSER`, `MYSQLDATABASE`. Password can be specified either in the
-`MYSQLPWD` environment variable or provided at the password prompt.
-
-Note that the various target-profile params described in the previous section
-are also applicable in direct connect mode.
-
-## Schema Conversion
+---
+layout: default
+title: MySQL
+parent: Data Type Conversion
+nav_order: 2
+---
+
+# Schema migration for MySQL
+{: .no_toc }
+
+Spanner migration tool makes some assumptions while performing data type conversion from MySQL to Spanner.
+There are also nuances to handling certain specific data types. These are captured below.
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+## Data Type Mapping
 
 The Spanner migration tool maps MySQL types to Spanner types as follows:
 
 | MySQL Type                                        | Spanner Type    | Notes                           |
-| ------------------------------------------------- | --------------- | ------------------------------- |
```

Comment:
A hyperlink to the text that explains what c,t etc. means will help

Reply:
It was in a paragraph below, converted the paragraph to a table for clarity.